### ISSUE TITLE
network seems study nothing

### ISSUE THREAD

**Issue by GaoQiyu** [voxelmorph/voxelmorph#200]
**Task** (what are you trying to do/register?)

aorta CT patch

**What have you tried**

vm1 and vm2, also try to replace unet core with more deeper unet

**Details of experiments**

input fixed /moved data : 168*128*256(xyz) aorta patch (CT)
net: 
cvpr2018_net(pytorch version)
unet_core:
    vm1/vm2/res unet
loss:
ncc loss

I trained the model on 245 aorta patch CT data, and used ncc loss only,.
In preprocess, I set a mediastinum window to clip the HU between -160 and 240,  then sent the fixed and moved data to model.
 In the begining the cc matrix is bery big, then I found there were lots of value in the cc matrix  is bigger than 40000, I think it is the problem of 1e-6 in denominator. So I remove the 1e-6 and set Nan in cc matrix to 0. And then the 1-torch.mean(cc) is always 0.9xxx, it seems that the network learned nothing.
I evaluate the model using dice coefficient, and the value never change even if I train this model a long time.
I am really very confused.
